pkg/cmd/cli/cmd/idle.go

```diff
@@ -0,0 +1,557 @@
+package cmd
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	utilerrors "github.com/openshift/origin/pkg/util/errors"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	osclient "github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+	deployclient "github.com/openshift/origin/pkg/deploy/client/clientset_generated/internalclientset/typed/core/unversioned"
+	unidlingapi "github.com/openshift/origin/pkg/unidling/api"
+	utilunidling "github.com/openshift/origin/pkg/unidling/util"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/meta"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	clientset "k8s.io/kubernetes/pkg/client/unversioned/adapters/internalclientset"
+	"k8s.io/kubernetes/pkg/controller"
+	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/util/strategicpatch"
+)
+
+const (
+	idleLong = `
+Idle scalable resources.
+
+This command idles the provides list of resources by scaling scalable resources (e.g. RCs and
```

Comment:
Typo.  Don't use "scalables", sounds awkward.  Describe what idling is first, then describe what can be idled, then describe what happens on idling.